packages/fxa-auth-server/lib/senders/emails/storybook-email.ts

```diff
@@ -5,20 +5,29 @@
 export interface StorybookEmailArgs {
   template: string;
   layout: string;
+  acceptLanguage: string;
   doc?: string;
   variables: Record<string, any>;
 }
 
+export const commonArgs = {
+  link: 'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
```

Comment:
I see `fxa-cad-reminder-first` in `link`, `privacyUrl`, and `supportUrl` here. Anything template-specific should live in that template, so `link` should be moved to `cadReminderFirst` since it goes to `/connect_another_device`. We have a few options for `privacyUrl` and `supportUrl` which we appear to use across templates...

1) Change them to `#`. Since this is just Storybook, it's really just serving as a visual tool for us - if we ever change any of the parameters on the URL here, or the URL in general, the default vars we set in Storybook here is going to be out of date.
2) We can strip out the CAD-reminder params, in this case `utm_campaign=fx-cad-reminder-first`, and only use the link + `utm_medium=email` + `utm_content`. We might want to leave a comment in the code just to be explicitly clear that other parameters exist in production, so `privacyUrl` for example would become: ```https://www.mozilla.org/privacy?utm_medium=email&utm_content=fx-privacy` // in production, other `utm` parameters may exist```
3) turn `commonArgs` into a function that accepts the name of the template, which would turn `privacyUrl` for example to something like: `https://www.mozilla.org/privacy?utm_medium=email&utm_campaign=fx-${emailName}`
4) We can strip out the params entirely. `privacyUrl` would become `https://www.mozilla.org/privacy` with a similar comment in the code about UTM params existing in production. **I think this is what I'd do personally,** it gives the Storybook viewer a chance to see the actual URL but without us having to think about/manage the UTM params in multiple places. If someone wants to see what they will actually end up being, they can use `yarn write-emails`.